src/wasm-type.h

```diff
@@ -119,14 +122,25 @@ constexpr Type i64 = Type::i64;
 constexpr Type f32 = Type::f32;
 constexpr Type f64 = Type::f64;
 constexpr Type v128 = Type::v128;
+constexpr Type funcref = Type::funcref;
 constexpr Type anyref = Type::anyref;
+constexpr Type nullref = Type::nullref;
 constexpr Type exnref = Type::exnref;
 constexpr Type unreachable = Type::unreachable;
 
 unsigned getTypeSize(Type type);
 FeatureSet getFeatures(Type type);
 Type getType(unsigned size, bool float_);
+bool isLeftSubTypeOfRight(Type left, Type right);
```

Comment:
> can be just `isSubType` - seems natural that it asks if the first param is a subtype of the second, I think?

Done. I named this function `isLeftSubTypeOfRight` because I thought it would be confusing which one is a subtype and which is a supertype... but maybe it'd be not very confusing after all.

> Btw, it's not obvious to me from the name if this is strict or not (is A a subtype of A). Is this obvious in type theory? (in set theory it would probably mean e.g. A is a subset of A). Perhaps a comment is worthwhile.

I think so - I mean, when you say A is a subtype of B, this means `A <: B` or `A <= B`, depending on the notation. To denote a subtype that's not the same as the given type, people usually use 'strict subtype' or something. The [wikipedia article on subtyping](https://en.wikipedia.org/wiki/Subtyping) seems to agree on this. I'll add comment clarifying the subtype here includes the type itself.

> (side issue unrelated to this PR, perhaps these toplevel methods should all be static and on `Type`?)

Done. I also found there are some `Type`-related methods that are outside of `Type` class and moved them into the class in #2556.